server/tests/e2e/admin.e2e-spec.ts

```diff
@@ -826,6 +852,81 @@ describe('Admin', () => {
             }));
     });
 
+    describe('GET /api/admin/reports', () => {
+        it('should return a list of reports', async () => {
+            const reports = await get({
+                url: `admin/reports`,
+                status: 200,
+                token: adminUserToken
+            });
+            expect(reports.body).toBeValidPagedDto(ReportDto);
+            expect(reports.body).toHaveProperty('response');
+
+            /*
+				data: "report",
+				type: ReportType.MAP_REPORT,
+				category: ReportCategory.INAPPROPRIATE_CONTENT,
+				message: "report message",
+				resolved: false,
+				resolutionMessage: "",
+				submitterID: user1.id,
+			*/
+
+            const count = await global.prisma.report.count();
+            expect(reports.body.response[0].data).toBe('report');
+            //expect(reports.body.responce[0].type).toBe(ReportType.MAP_REPORT);
+            expect(reports.body.response[0].category).toBe(ReportCategory.INAPPROPRIATE_CONTENT);
+            expect(reports.body.response[0].message).toBe('report message');
+            expect(reports.body.response[0].resolved).toBe(false);
+            expect(reports.body.response[0].resolutionMessage).toBe('');
+            expect(reports.body.response[0].submitterID).toBe(user1.id);
+        });
+
+        it('should limit the result set when using the take query param', async () => {
+            const reports = await get({
+                url: `admin/reports`,
+                status: 200,
+                query: {
+                    take: 1
+                },
+                token: adminUserToken
+            });
+            expect(reports.body).toBeValidPagedDto(ReportDto);
+            expect(reports.body).toHaveProperty('response');
+            expect(reports.body).toHaveProperty('returnCount');
+            expect(reports.body.returnCount).toBe(1);
+            expect(reports.body.response[0].data).toBe('report');
+        });
+
+        it('should skip some of the result set when using the skip query param', async () => {
+            const reports = await get({
+                url: `admin/reports`,
+                status: 200,
+                query: {
+                    skip: 1
+                },
+                token: adminUserToken
+            });
+            expect(reports.body).toBeValidPagedDto(ReportDto);
+            expect(reports.body).toHaveProperty('response');
+            expect(reports.body).toHaveProperty('returnCount');
+            expect(reports.body.returnCount).toBe(1);
+            expect(reports.body.response[0].data).toBe('report2');
+        });
```

Comment:
These should be generic `skipTest` and `takeTest`s. The stuff in `generic-e2e-tests.util.ts` exists for a reason!